### DISCUSSION TITLE
Update 2.8.0 Is Now In Development!

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#65] [Announcements]
Hey guys, I am glad to announce that development for **the next major update is underway**!

**2.8.0 features** the completion of **simulation mode**, a **semi-realistic** version of the **progression system that uses humidity, temperature**, and many other things to create weather.

It features **more equipment you can use**, **new radar mechanics**, **vehicles**, **more storm types**, somewhat **better visuals**, and much more!

Feel free send me suggestions/bugs/exploits in **Issues**, visit **Projects** to see how much I did so far or what I have planned, or visit me in my [discord ](https://discord.gg/trNQFjZ26w) to get the latest news, chat with me and other people, or watch me develop the update when I stream there.

Have a great day guys!

**Comment by GGlitchX** [comment]
Super excited, quick question though, are frontal systems and pressure systems going to be added, and if so, when?

**Reply by Mrbt0907** [reply]
When I start finishing simulation mode, I might add it in since you did give me a good way to do it.

**Reply by GGlitchX** [reply]
Ah yeah, I did try to keep my way in a way its hopefully simpler to code into than actually using full on scientific formulas but still fairly accurate.

**Reply by oriolprogamer712** [reply]
when will you put the release of the mod.

**Reply by Mrbt0907** [reply]
> when will you put the release of the mod.

The full release of the weather2 update as a whole will finish once I have added all the features I wanted to add in.

The mod will remain in an alpha stage while I am still changing things around. The beta will start once I get everything done and I need testing and feedback as I polish the mod for the full release.
